source/style-guide/information-types/prototypes/reference.txt

```diff
@@ -27,6 +27,31 @@ contents, often called a short description. The short description
 explains a little about how the reference material is used, and may link
 to a concept or task page.
 
+Compatibility 
+-------------
+
+Optional. 
+
+A list of MongoDB deployment types that support the referenced method, operator, 
+or database command: 
+
+- `MongoDB Atlas <https://docs.atlas.mongodb.com/>`_: The fully managed service 
+  for MongoDB deployments in the cloud
+
+.. note:: 
+
+   If the command is supported on Atlas, specify whether it's available in all 
+   clusters, has limited support for some clusters, or is not available in 
+   specific clusters. If there are limitations to which clusters support the 
+   command, direct users to the 
+   :atlas:`Unsupported Commands </unsupported-commands>` page.
+
+- `MongoDB Community <https://www.mongodb.com/try/download/community>`_: The 
+  source-available, free-to-use, and self-managed version of MongoDB
+
+- `MongoDB Enterprise <http://www.mongodb.com/products/mongodb-enterprise-advanced>`_: 
+  The subscription-based, self-managed version of MongoDB
```

Comment:
Suggestion:

A couple suggestions:

1) I think Enterprise should be listed before Community since that would give it a better chance of being seen.

2) I think it would be helpful to identify whether this is the content that writers should use verbatim. Most prototype sections only describe the content or provide hypothetical examples rather than directly provide the content.

For driver docs, we've moved this [compatibility CTA to docs-shared](https://github.com/10gen/docs-shared/blob/main/dbx/atlas-connect-compatible-deployment.rst) so that we don't have to worry about reviewing/updating when included in an unrelated PR.

Reply:
In response to your first point, the list items here come from includes (at least in the server docs). To tie in your second note, I can point writers to these includes, as well as the drivers team's docs-shared CTA, in a tip below the list!

Reply:
Looks good to me, thanks!